File: backend/domain/dto/DiagnosisResponseDTO.java
package com.example._th_project.domain.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Builder
public class DiagnosisResponseDTO {
    private Long diagnosisId;
    private byte[] diseaseImg;
    private String petName;
    private String stage;
    private Double riskScore;
    private String diseases;
    private String diseasesContent;
    private String treatment;
    private List<String> medications;

    @Builder
    public  DiagnosisResponseDTO(Long diagnosisId, byte[] diseaseImg, String petName, String stage, Double riskScore, String diseases, String diseasesContent, String treatment, List<String> medications){
        this.diagnosisId = diagnosisId;
        this.diseaseImg = diseaseImg;
        this.petName = petName;
        this.stage = stage;
        this.riskScore = riskScore;
        this.diseases = diseases;
        this.diseasesContent = diseasesContent;
        this.treatment = treatment;
        this.medications = medications;
    }
}

File: frontend/android/model/DiagnosisResponseDTO.kt
package com.project.meongnyangcare.model

import com.google.gson.annotations.SerializedName

data class DiagnosisResponseDTO(
    @SerializedName("diagnosisId") val diagnosisId: Long,
    @SerializedName("diseaseImg") val diseaseImg: String,
    @SerializedName("petName") val petName: String,
    @SerializedName("stage") val stage: String,
    @SerializedName("riskScore") val riskScore: Double,
    @SerializedName("diseases") val diseases: String,
    @SerializedName("diseasesContent") val diseasesContent: String,
    @SerializedName("treatment") val treatment: String,
    @SerializedName("medications") val medications: List<String>
)
